api/PclusterApiHandler.py

```diff
@@ -132,15 +132,16 @@ def authenticate(group):
     except Exception as e:
         return abort(401)
 
-    if (group != "guest") and (group not in set(decoded.get(USER_ROLES_CLAIM, []))):
+    jwt_roles = set(decoded.get(USER_ROLES_CLAIM, []))
+    groups_granted = groups.intersection(jwt_roles)
+    if ("guest" not in groups) and len(groups_granted) == 0:
         return abort(403)
 
-
-def authenticated(group="user"):
+def authenticated(groups={"user"}):
```

Comment:
question(non-blocking): Why did you choose only user as default? using admin and user would make more sense in my opinion, and possibly would remove some code in the app.py

Reply:
I just kept the old behavior, but there are no places right now where the "default" is used (all invocations to `authenticated` are with parameters). Since I dislike these implicit behaviors, what I propose is to get rid entirely of the default parameter, and specify always a set of groups. What do you think? @eantonin

Reply:
Yeah, I think your proposal is the better solution in this case

Reply:
Fixed!